### ISSUE TITLE
Allow configurable unprivileged port in helm chart

### ISSUE THREAD

**Issue by FischerLGLN** [developmentseed/titiler#485]
Helm-installed Titiler is not running in hardened environments like openshift, if containers run listening on port 80.

See yaml in: https://github.com/developmentseed/titiler/blob/master/deployment/k8s/titiler/templates/deployment.yaml#L36
```yaml
ports:
  - name: http
    containerPort: 80
    protocol: TCP
```
Solution:

env definition sets the containerPort
```yaml
# Already user configurable in values.yaml
env:
  PORT: 8080
```
```yaml
# Required change in deployment.yaml
ports:
  - name: http
    containerPort: {{ .Values.env.PORT }}
    protocol: TCP
```

After this change, titiler runs in openshift restrictive mode

**Comment by vincentsarago** [comment]
Wave @FischerLGLN 
feel free to open a Pull request 🙏

**Comment by FischerLGLN** [comment]
@vincentsarago Thanks, I will do that soon!

**Comment by FischerLGLN** [comment]
Wave @vincentsarago PR is online :tada:

**Comment by vincentsarago** [comment]
closed by #486
